src/main/java/uk/gov/hmcts/probate/services/businessdocuments/model/BulkScanCoverSheet.java

```diff
@@ -0,0 +1,113 @@
+package uk.gov.hmcts.probate.services.businessdocuments.model;
+
+import org.hibernate.validator.constraints.NotBlank;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonRootName;
+
+@JsonRootName(value = "bulkScanCoverSheet")
+public class BulkScanCoverSheet implements BusinessDocument {
+		
+	public static final String DEFUALT_TITLE = "Download Cover Sheet";
+	public static final String DEFUALT_APPLICANT_ADDRESS_INTRO = "Your address";
+	public static final String DEFUALT_CASE_REFERENCE_INTRO = "Your unique reference\nnumber is";
+	public static final String DEFUALT_SUBMIT_ADDRESS_INTRO = 
+			"Please send this cover sheet along with your document(s) to the address shown below";
+
+	private static final char CASE_REFERENCE_SEPARATOR_CHAR = '-';
+	private static final int CASE_REFERENCE_SEPARATOR_INTERVAL_NUMBER = 4;
+	
+	@NotBlank
+    @JsonProperty("title")
+    private String title = DEFUALT_TITLE;
+	
+	@NotBlank
+    @JsonProperty("applicantAddressIntro")
+    private String applicantAddressIntro = DEFUALT_APPLICANT_ADDRESS_INTRO;
+
+	@NotBlank
+    @JsonProperty("applicantAddress")
+    private String applicantAddress;
+
+	@NotBlank
+    @JsonProperty("caseReferenceIntro")
+    private String caseReferenceIntro = DEFUALT_CASE_REFERENCE_INTRO;
+	
+	@NotBlank
+    @JsonProperty("caseReference")
+    private String caseReference;
+
+	@NotBlank
+    @JsonProperty("submitAddressIntro")
+    private String submitAddressIntro = DEFUALT_SUBMIT_ADDRESS_INTRO;
+	
+	@NotBlank
+    @JsonProperty("submitAddress")
+    private String submitAddress;
+		
+	public String getTitle() {
+		return title;
+	}
+
+	public void setTitle(String title) {
+		this.title = title;
+	}
+
+	public String getApplicantAddressIntro() {
+		return applicantAddressIntro;
+	}
+
+	public void setApplicantAddressIntro(String applicantAddressIntro) {
+		this.applicantAddressIntro = applicantAddressIntro;
+	}
+
+	public String getCaseReferenceIntro() {
+		return caseReferenceIntro;
+	}
+
+	public void setCaseReferenceIntro(String caseReferenceIntro) {
+		this.caseReferenceIntro = caseReferenceIntro;
+	}
+
+	public String getSubmitAddressIntro() {
+		return submitAddressIntro;
+	}
+
+	public void setSubmitAddressIntro(String submitAddressIntro) {
+		this.submitAddressIntro = submitAddressIntro;
+	}
+	
+	public String getApplicantAddress() {
+		return applicantAddress;
+	}
+
+	public void setApplicantAddress(String applicantAddress) {
+		this.applicantAddress = applicantAddress;
+	}
+
+	public String getCaseReference() {
+		return caseReference;
+	}
+
+	public void setCaseReference(String caseReference) {
+		this.caseReference = addCaseReferenceHyphens(caseReference);
+	}
+
+	public String getSubmitAddress() {
+		return submitAddress;
+	}
+
+	public void setSubmitAddress(String submitAddress) {
+		this.submitAddress = submitAddress;
+	}
+	
+	private String addCaseReferenceHyphens(String number) {
```

Comment:
Can't see a unit test that is called via setCaseReference when number contains hyphens or not.